app/controllers/api/pages_controller.rb

```diff
@@ -58,6 +58,17 @@ def similar
     render :index, format: :json
   end
 
+  def total_donations
+    @page = Page.find(params[:page_id])
+    @amount = if @page.campaign.blank?
+                @page.total_donations
+              else
+                @page.campaign.donations_count
+              end
+    total_donations = FundingCounter.convert(currency: params[:currency], amount: @amount)
+    render json: { total_donations: total_donations.to_s }
```

Comment:
@Tuuleh can we also send back some ancillary data, like the page name and the campaign name?

Reply:
also the goal amount?

Reply:
Yes - the goal amount hasn't been implemented yet so I'm not sending it back yet, we'll need to add it (it's a part of the stuff I'm doing right now after I figure the best way to have the donations thermometer alongside the actions thermometer). What all do we want to send back? Campaign name + id (if there was a campaign), page titles for all the related pages corresponding with their languages?